app/controllers/project/settings/donations.js

```diff
@@ -0,0 +1,67 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
```

Comment:
You should be able to use the `ember-route-action-helper` if you don't want to go through the controller.  Although, I don't see anything wrong with using a controller in this case.